alibi_detect/cd/chisquare.py

```diff
@@ -112,8 +112,8 @@ def feature_score(self, X_ref: np.ndarray, X: np.ndarray) -> Tuple[np.ndarray, n
         p_val = np.zeros(self.n_features, dtype=np.float32)
         dist = np.zeros_like(p_val)
         for f in range(self.n_features):  # apply Chi-Squared test
-            n_ref, n_obs = X_ref_count[f].sum(), X_count[f].sum()
-            dist[f], p_val[f] = chisquare(X_count[f], f_exp=X_ref_count[f] * n_obs / n_ref)
+            contingency_table = np.vstack((X_ref_count[f], X_count[f]))
+            dist[f], p_val[f], _, _ = chi2_contingency(contingency_table)
```

Comment:
Leaving `lambda_=None` applies Pearson's Chi^2 test. Do we think it is worth allowing for other tests as well by specifying `lambda_` at init? See https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.chi2_contingency.html.

Reply:
Just looked into this a little and after looking at [this paper](https://hal.archives-ouvertes.fr/hal-00514341/document) I'm not sure that it's worth the added complexity. Pearson's generally considered best and also the conditions under which it suitable and accurate is best understood. Speaking of which, I wonder if it might be worth adding warnings when such conditions (see p2 of aforementioned paper) are not met?

Reply:
Good idea. Might even be worth returning it as well with the drift prediction? Could be helpful for downstream applications.

Reply:
The warning sounds like a nice idea (provided we link to justification in the docs) and agree that makes sense to include this info in the return too. I would make it a separate PR though to keep things more atomic.

A more general conversation we can have in the future is if we should support more tests as there are a plethora of frequentist tests in the literature.